textrender.cpp

```diff
@@ -76,7 +76,11 @@ TextRender::TextRender(QQuickItem *parent) :
     iShowBufferScrollIndicator = false;
 
     iFont = QFont(sUtil->fontFamily(), sUtil->fontSize());
+#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
     iFont.setStyleHint(QFont::Monospace, QFont::StyleStrategy(QFont::PreferDefault | QFont::ForceIntegerMetrics));
+#else
+    iFont.setStyleHint(QFont::Monospace, QFont::StyleStrategy(QFont::PreferDefault));
```

Comment:
Prefer default would be the default so does this do anything? For second thing is even the earlier ForceIntegerMetrics doing anything since there is QFontMetrics used for sizes as suggested by the deprecation notice?